### ISSUE TITLE
(crispy-heretic) continuing recording demo not working 

### ISSUE THREAD

**Issue by thom-wye** [fabiangreffrath/crispy-doom#691]
### Background

Version of Crispy Doom: daily build 20201027

Operating System and version: Windows 10

Game: Heretic

Any loaded WADs and mods (please include full command line): heretic.wad

### Bug description

Observed behavior: using the parameters -playdemo  and -record simultaneously starts a new demo (e1m1 spawn without any further parameters)

Expected behavior: the crispy-doom behaviour where the two parameters together fast-forwards you to the end of the existing demo

I didn't know I'd need -maxdemo )

**Comment by fabiangreffrath** [comment]
Yep, Crispy Heretic is still quite a bit behind Crispy Doom in terms of demo features.
